### PR TITLE
OpenSSL activate legacy support

### PR DESCRIPTION
**Pull request by QU35T-code**
# Description

* Openssl configuration issue that prevents SSL use with evil-winrm.
* MD4 support

# Related issues

* https://forum.hackthebox.com/t/evil-winrm-error-on-connection-to-host/257342/9

### CODE REVIEW

File: sources/install/package_ad.sh
Comment (ShutdownRepo) [inline sources/install/package_ad.sh:1]: wouldn't it be best to have a file in assets and copy it at build?

Comment (QU35T-code) [reply sources/install/package_ad.sh:1]: The problem is that the base file could be modified by an openssl update and that could break everything, hence the need to add the content.
If you want, I can put the fix in a file and do a : `cat patch >> /etc/ssl/openssl.conf`

Comment (ShutdownRepo) [reply sources/install/package_ad.sh:1]: go for cat >>
cleaner that way imo

Comment (QU35T-code) [reply sources/install/package_ad.sh:1]: Done

**Review by ShutdownRepo — APPROVED** [review]

**Review by ShutdownRepo — APPROVED** [review]

**Pull request merged**